apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Config.java

```diff
@@ -163,11 +163,14 @@
          * Get profile task list interval
          */
         public static int GET_PROFILE_TASK_INTERVAL = 20;
-
         /**
          * Get agent dynamic config interval
          */
         public static int GET_AGENT_DYNAMIC_CONFIG_INTERVAL = 20;
+        /**
+         * If true, skywalking agent will enable periodically resolving DNS to update receiver service addresses. Otherwise disable
+         */
+        public static boolean DNS_PERIOD_RESOLVE_ACTIVE = false;
```

Comment:
```suggestion
        public static boolean IS_RESOLVE_DNS_PERIODICALLY = false;
```
I think the current naming style is like this.

Reply:
fixed